java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/PythonApiIT.java

```diff
@@ -0,0 +1,187 @@
+        }
+
+        @Test
+        void shouldBatchWriteOneFile() throws IOException, InterruptedException {
+            // Given
+            sleeper.localFiles(tempDir)
+                    .createWithNumberedRecords("file.parquet", LongStream.range(0, 100));
+
+            // When
+            sleeper.pythonApi(tempDir)
+                    .ingest().batchWrite("file.parquet")
+                    .invokeTask().waitForJobs();
+
+            // Then
+            assertThat(sleeper.directQuery().allRecordsInTable())
+                    .containsExactlyElementsOf(sleeper.generateNumberedRecords(LongStream.range(0, 100)));
+            assertThat(sleeper.tableFiles().active()).hasSize(1);
+        }
+
+        @Test
+        void shouldIngestTwoFilesFromS3() throws IOException, InterruptedException {
+            // Given
+            sleeper.sourceFiles()
+                    .createWithNumberedRecords("file1.parquet", LongStream.range(0, 100))
+                    .createWithNumberedRecords("file2.parquet", LongStream.range(100, 200));
+
+            // When
+            sleeper.pythonApi(tempDir)
+                    .ingest().fromS3("file1.parquet", "file2.parquet")
+                    .invokeTask().waitForJobs();
+
+            // Then
+            assertThat(sleeper.directQuery().allRecordsInTable())
+                    .containsExactlyElementsOf(sleeper.generateNumberedRecords(LongStream.range(0, 200)));
+            assertThat(sleeper.tableFiles().active()).hasSize(1);
+        }
+
+        @Test
+        void shouldIngestDirectoryFromS3() throws IOException, InterruptedException {
+            // Given
+            sleeper.sourceFiles()
+                    .createWithNumberedRecords("test-dir/file1.parquet", LongStream.range(0, 100))
+                    .createWithNumberedRecords("test-dir/file2.parquet", LongStream.range(100, 200));
+
+            // When
+            sleeper.pythonApi(tempDir)
+                    .ingest().fromS3("test-dir")
+                    .invokeTask().waitForJobs();
+
+            // Then
+            assertThat(sleeper.directQuery().allRecordsInTable())
+                    .containsExactlyElementsOf(sleeper.generateNumberedRecords(LongStream.range(0, 200)));
+            assertThat(sleeper.tableFiles().active()).hasSize(1);
+        }
+
+        @Test
+        void shouldBulkImportFilesFromS3() throws IOException, InterruptedException {
+            // Given
+            sleeper.updateTableProperties(tableProperties ->
+                    tableProperties.set(TableProperty.BULK_IMPORT_MIN_LEAF_PARTITION_COUNT, "1"));
+            sleeper.sourceFiles()
+                    .createWithNumberedRecords("file1.parquet", LongStream.range(0, 100))
+                    .createWithNumberedRecords("file2.parquet", LongStream.range(100, 200));
+
+            // When
+            sleeper.pythonApi(tempDir)
+                    .bulkImport().emrServerless("file1.parquet", "file2.parquet")
```

Comment:
It's hard to tell that this is the same as `fromS3` but for EMR Serverless. Can we give them matching names so that we can tell they'll ingest by queue based with a file that's already in S3?